src/main/java/org/fcrepo/audit/InternalAuditor.java

```diff
@@ -231,7 +226,18 @@ public void createAuditNode(final FedoraEvent event) throws IOException {
     }
 
     @VisibleForTesting
+    protected String getEventPath(final String eventID) {
+        if (!eventID.startsWith("urn:uuid:")) {
+            throw new IllegalArgumentException("Event ID must be a 'urn:uuid:'" + eventID);
+        }
+        return pathMinter.get(eventID.substring("urn:uuid:".length()));
```

Comment:
W/R/T `fcrepo-audit-triplestore` I would prefer to keep that independent of `fcrepo-kernel-api`. At present, the kernel API is pretty "heavy" -- it requires guava, jena, jcr and servlet-api. I would really rather not load all of that into OSGi just for two lines of code.